Library/Formula/muparserx.rb

```diff
@@ -0,0 +1,44 @@
+class Muparserx < Formula
+  homepage "http://muparser.beltoforion.de/"
+
+  stable do
+    url "https://muparserx.googlecode.com/files/muparserx_v2_1_6.zip"
+    sha1 "f5b6ed13594c14f408168f6c0111ba4078e4b12f"
+  end
+  head "http://muparserx.googlecode.com/svn/trunk/"
+
+  depends_on "cmake" => :build
+
+  resource "CMakeLists.txt" do
+    url "http://leezii.com/CMakeLists.txt"
```

Comment:
@mistydemeo When I put version "2.1.6", I got one audit problem
* Stable: version 2.1.6 is redundant with version scanned from URL

What do you mean "It also looks like this isn't used?"